articles/azure-large-instances/workloads/epic/onboarding-requirements.md

```diff
@@ -0,0 +1,190 @@
+
+3. Set up ALI for Epic solution based on your system requirements and take a system backup.  
+4. Take an OS backup.  
+5. Set up volume groups. (See [Create a volume group](create-a-volume-group.md).)  
+6. Set up a storage snapshot, backup, and data offload. (For detailed steps, see [Azure Large Instances NETAPP storage data protection with Azure CVO](ali-netapp-with-cvo.md)).
+
+> [!Note]
+> A storage snapshot should only be set up after all data-intensive work (for example, Endian conversions) are complete in order to avoid creating unnecessary snapshots while build work is in progress
+
+The Azure subscription you use for ALI deployments is already registered with the ALI resource provider by the Microsoft Operations team during the provisioning process.
+If you don't see your deployed Azure Large Instances under your subscription, register the resource provider with your subscription. For more information, see [Register the ALI resource provider](register-the-ali-resource-provider.md).
+
+### Enable ExpressRoute Fast Path
+
+Before you begin, install the latest version of the Azure resource manager power shell cmdlets, at least 4.0 or later.
+For more information about installing the power shell cmdlets, see [How to install Azure Powershell](https://learn.microsoft.com/powershell/azure/install-azure-powershell?view=azps-10.0.0).
+
+For more information, see these resources:
+
+* [Azure ExpressRoute overview](https://azure.microsoft.com/products/expressroute/)
+
+* [How to create a connection between your VPN Gateway and ExpressRoute circuit](https://learn.microsoft.com/shows/azure/expressroute-how-to-create-connection-between-your-vpn-gateway-expressroute-circuit) 
+
+
+* [How to set up Microsoft peering for your ExpressRoute circuit](https://learn.microsoft.com/shows/azure/expressroute-how-to-set-up-microsoft-peering-your-expressroute-circuit)
+
+### Authorizing  
+
+Ensure you have an authorization key for the express route (ER) circuit used for virtual gateway connection to ER circuit. 
+Also obtain ER circuit resource ID.
+
+If you don’t have this information, obtain the details from the circuit owner (these details are usually provided by the Microsoft team as part of provisioning request completion.
+Reach out to
+<a href=mailto:"AzureBMISupportEpic@microsoft.com">Microsoft support</a> in case of any inconsistencies).  
+
+### Declare variables
+
+This example declares the variables using the values for this exercise.
+Replace the values with your subscription values.
+
+```azurecli
+$Sub1 = "Replace_With_Your_Subcription_Name"  
+
+$RG1 = "TestRG1"  
+
+$Location1 = "East US"  
+
+$GWName1 = "VNet1GW"  
+
+$Authkey = “Express route circuit auth key”  
+
+$Sub1 = "Replace_With_Your_Subcription_Name"  
+
+$RG1 = "TestRG1"  
+
+$Location1 = "East US"  
+
+$GWName1 = "VNet1GW"  
+
+$Authkey = “Express route circuit auth key”  
+```
+  
+### Connect to your account
+
+```azurecli
```

Comment:
These are Powershell commands and not AzureCLI

@vasthomsft I believe, we should update these commands to Az modules instead of AzureRM Modules.

Reply:
agreed @makotrik on using Az modules. @ianmsgit , @ryellajo12 , Let's use the following instead. @makotrik , please help validate the accuracy of these commands

Declare Variables

$Sub1 = "Replace_With_Your_Subscription_Name"
$RG1 = "Your_Resource_Group_Name"
$CiruitName="ExpressRoute Circuit Name"
$Location1="Location_Name" #Example: "East US"
$GWName1="VNET_Gateway_Name"
$ConnectionName=”ER Gateway Connection Name”
$Authkey="ExpressRoute Circuit's Authorization Key" 

Connect to your account.

Check the subscription for the account
Get-AzSubscription

select the subscription that you want to use
Select-AzSubscription -SubscriptionId $Sub1 

Enable ER FastPath on the gateway connection.

Declare Variable for ExpressRoute circuit 
$circuit = Get-AzExpressRouteCircuit -Name $CiruitName -ResourceGroupName $RG1

Declare Variable for Gateway 
$gw = Get-AzVirtualNetworkGateway -Name $GWName1 -ResourceGroupName $RG1

Enable MSEEv2 using "ExpressRouteGatewayBypass" flag
$connection = New-AzVirtualNetworkGatewayConnection -Name $ConnectionName -ResourceGroupName $RG1 -ExpressRouteGatewayBypass -VirtualNetworkGateway1 $gw -PeerId $circuit.Id -ConnectionType ExpressRoute -Location $Location1 -AuthorizationKey $Authkey

Reply:
@makotrik have you been able to validate the accuracy of the new commands @vasthomsft added above?

Reply:
Please find the corrected steps here...

Declare Variables

$Sub = "Replace_With_Your_Subscription_ID"
$RG = "Your_Resource_Group_Name"
$CircuitName="ExpressRoute Circuit Name"
$Location="Location_Name" #Example: "East US"
$GWName="VNET_Gateway_Name"
$ConnectionName=”ER Gateway Connection Name”
$Authkey="ExpressRoute Circuit Authorization Key"

Connect to your account.
Login-AzAccount


Check if the intended subscription is listed in the output of below command.
Get-AzSubscription


Select the subscription that you want to use
Select-AzSubscription -SubscriptionId $Sub

Enabling ER FastPath for the gateway connection.
$Circuit = Get-AzExpressRouteCircuit -Name $CircuitName -ResourceGroupName $RG
$GW = Get-AzVirtualNetworkGateway -Name $GWName -ResourceGroupName $RG


Enable MSEEv2 using "ExpressRouteGatewayBypass" flag
$connection = New-AzVirtualNetworkGatewayConnection -Name $ConnectionName -ResourceGroupName $RG -ExpressRouteGatewayBypass -VirtualNetworkGateway1 $GW -PeerId $Circuit.Id -ConnectionType ExpressRoute -Location $Location -AuthorizationKey $Authkey